src/Restaurants.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export default function Restaurants({ restaurants }) {
+  return (
+    <ul>
+      {restaurants.map((restaurant) => (
+        <li key={restaurant.id}>{restaurant.name}</li>
```

Comment:
구조 분해 할당으로 처리해줘도 좋을 것 같네요~

```suggestion
      {restaurants.map(({ id, name }) => (
        <li key={id}>{name}</li>
```

Reply:
훨씬 가독성이 좋은거같아요! 반영했습니다